scripts/install/usr/libexec/cortx-motr/motr-server

```diff
@@ -199,11 +199,12 @@ m0_server()
     # show actual exec commands in log file
     set -x
 
+    addb_opts+=" -r ${MOTR_MOD_ADDB_RECORD_SIZE}"
```

Comment:
This is WRONG, but the result is accidentally right. Please use:
```
addb_opts=" -r ${MOTR_MOD_ADDB_RECORD_SIZE}"
```

"+=" means to append.
But this is the first time appearance of this variable.

Reply:
Corrected 2bf6554